AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -605,10 +605,6 @@ extension MeetingModel: VideoTileObserver {
             } else {
                 videoModel.addRemoteVideoTileState(tileState, completion: {
                     if self.activeMode == .video {
-                        // If the video is not currently being displayed, pause it
-                        if !self.videoModel.isRemoteVideoDisplaying(tileId: tileState.tileId) {
-                            self.currentMeetingSession.audioVideo.pauseRemoteVideoTile(tileId: tileState.tileId)
-                        }
```

Comment:
Discussed offline, will need to check if videos in other pages are in resumed state or still subscribed, if so, it may occupy unnecessary CPU/memory resources bc videos are not presented.

Reply:
Verified that when we are on the first page, videos on other pages(not present) are not added to the list of videoSubscriptionConfiguration.